environment/commands/dump-eth-headers/index.js

```diff
@@ -0,0 +1,31 @@
+const Web3 = require('web3');
+const Path = require('path');
+const fs = require('fs').promises;
+
+class DumpETHHeaders {
+  static async execute({path, startBlock, endBlock, ethNodeUrl}) {
+    console.log('here')
+    let web3 = new Web3(ethNodeUrl);
+    console.log('there')
+    console.log(path)
+    console.log(endBlock)
+    for (let b = startBlock; b <= endBlock; b++) {
```

Comment:
Please add functionality of when `endBlock` is not specified to use the most recent block.

Reply:
This was not the upto date one, i'm doing some cleanup and undo changes in ethrelay (which was using the subscribe method)